docs/articles/networking.md

```diff
@@ -215,12 +218,17 @@ Four different options affect container domain name services.
     only look up `host` but also `host.example.com`.
     Use `--dns-search=.` if you don't wish to set the search domain.
 
-Regarding DNS settings, in the absence of either the `--dns=IP_ADDRESS...`
-or the `--dns-search=DOMAIN...` option, Docker makes each container's
-`/etc/resolv.conf` look like the `/etc/resolv.conf` of the host machine (where
-the `docker` daemon runs).  When creating the container's `/etc/resolv.conf`,
-the daemon filters out all localhost IP address `nameserver` entries from
-the host's original file.
+ *  `--dns-opt=OPTION...` — sets the options used by DNS resolvers
+    by writing an `options` line into the container's `/etc/resolv.conf`.
+    See documentation for `resolv.conf` for a list of valid options.  Use
+    `--dns-opt=''` if you don't wish to set options.
```

Comment:
If I *don't* add a `--dns-opt`, will there still be an `options` line written? I.e., is *omitting* `--dns-opt` different than adding `--dns-opt=''`? If there's no difference, then I'd drop the last line.